data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -188,9 +207,23 @@ def _save_full_csv(self):
 
     def _save_tile_csv(self):
         logger.info("Saving Tile Score CSV")
-        # TODO: check which are the columns we'll use
-        # Related to: https://github.com/usds/justice40-tool/issues/302
         score_tiles = self.score_county_state_merged[self.TILES_SCORE_COLUMNS]
+
+        # round decimals for tile columns
+        TILES_SCORE_FLOAT_COLUMNS = [
+            "Score E (percentile)",
+            "Score E (top 25th percentile)",
+            "Poverty (Less than 200% of federal poverty line)",
+            "Percent individuals age 25 or over with less than high school degree",
+            "Linguistic isolation (percent)",
+            "Unemployed civilians (percent)",
+            "Housing burden (percent)",
+        ]
+        decimals = pd.Series(
+            [2, 2, 2, 2, 2, 2, 2], index=TILES_SCORE_FLOAT_COLUMNS
```

Comment:
hm, this is a bit confusing - is it possible to do this as a map?

Reply:
Per our zoom convo, will make a constant at the top of the file with the float precision and generate the list using a generator (i.e. `[e] * n`)